infra/k8s/kops-weave/weave.yml

```diff
@@ -172,7 +172,7 @@ items:
             - sh
             - -ac
             - >
-              export GW=$(ip route | grep cni0 | awk '{print $7'}) &&
```

Comment:
OMG, not sure how this even worked, but I did verify the routes were there by inspecting the logs with `kubectl logs weave-id -c iproute-add`

Reply:
Because it "works" in some shells.